internal/exec/engine.go

```diff
@@ -82,7 +85,17 @@ func (e Engine) Run(stageName string) error {
 		e.Logger.Crit("failed to acquire config: %v", err)
 		return err
 	}
-
+	var haveConfig bool
+	// Check to see if we already have a cached config
+	if _, err := os.Stat(e.ConfigCache); err != nil {
+		haveConfig = true
+	}
```

Comment:
This should be done before `e.acquireConfig()` right? Since that function will always populate the cache. Also, let's use `os.IsNotExist`.

Probably also worth mentioning that this is a hack to ensure we only emit the journal message once and link to https://github.com/coreos/ignition/issues/950. :)